src/main/java/icbm/classic/CommonProxy.java

```diff
@@ -139,9 +140,9 @@ public void spawnSmoke(World world, Pos position, double v, double v1, double v2
 
     }
 
-    public void spawnAirParticle(World world, Pos position, double v, double v1, double v2, float red, float green, float blue, float scale, int age)
+    public void spawnAirParticle(World world, Pos position, double v, double v1, double v2, float red, float green, float blue, float scale, int ticksToLive)
     {
-
+        PacketSpawnAirParticle.sendToAllClients(world, position, v, v1, v2, red, green, blue, scale, ticksToLive);
```

Comment:
I'm concerned about the bandwidth of this and why we are not letting MC handle the logic.